tests/test_savings_kpi.py

```diff
@@ -0,0 +1,30 @@
+from unittest import TestCase
+from uuid import uuid4
+
+from d3a_interface.sim_results.kpi import SavingsKPI
+
+
+class TestSavingsKPI(TestCase):
```

Comment:
Since this is a new test, please avoid using `unittest.TestCase` and just rely on pytest's features. This is something that came up in the last two days during a conversation on slack.

Reply:
https://github.com/gridsingularity/d3a-interface/pull/223/commits/cb925bc2368e0d411295da61bf6b9e52467c9ae3